junit5/README.md

```diff
@@ -383,6 +385,103 @@ class MyTest {
 }
 ```
 
+#### Inheritance of test classes
+
+The `@WeldSetup` field can be defined in a superclass, but there can only be one `@WeldSetup` field in the class
```

Comment:
This is good catch and wasn't documented so thanks for that.
I wonder why we introduced this as opposed to picking the first field we discovered in the hierarchy...